src/command_modules/azure-cli-acr/README.rst

```diff
@@ -86,34 +86,39 @@ Update a container registry
         az acr update: Update a container registry.
     
     Arguments
-        --name -n [Required]: Name of container registry.
-        --resource-group -g : Name of resource group.
+        --name -n      [Required]: Name of container registry.
+        --admin-user-enabled -a  : Whether the admin user account is enabled.  Allowed values: false,
+                                   true.
+        --resource-group -g      : Name of resource group.
+        --storage-account-name -s: Name of an existing storage account.
+        --tags                   : Space separated tags in 'key[=value]' format. Use "" to clear
+                                   existing tags.
     
     Generic Update Arguments
-        --add               : Add an object to a list of objects by specifying a path and key value
-                              pairs.  Example: --add property.listProperty <key=value, string or JSON
-                              string>.
-        --remove            : Remove a property or an element from a list.  Example: --remove
-                              property.list <indexToRemove> OR --remove propertyToRemove.
-        --set               : Update an object by specifying a property path and value to set.  Example:
-                              --set property1.property2=<value>.
+        --add                    : Add an object to a list of objects by specifying a path and key value
+                                   pairs.  Example: --add property.listProperty <key=value, string or
+                                   JSON string>.
+        --remove                 : Remove a property or an element from a list.  Example: --remove
+                                   property.list <indexToRemove> OR --remove propertyToRemove.
+        --set                    : Update an object by specifying a property path and value to set.
+                                   Example: --set property1.property2=<value>.
     
     Global Arguments
-        --debug             : Increase logging verbosity to show all debug logs.
-        --help -h           : Show this help message and exit.
-        --output -o         : Output format.  Allowed values: json, jsonc, list, table, tsv.  Default:
-                              json.
-        --query             : JMESPath query string. See http://jmespath.org/ for more information and
-                              examples.
-        --verbose           : Increase logging verbosity. Use --debug for full debug logs.
+        --debug                  : Increase logging verbosity to show all debug logs.
```

Comment:
Sorry, sorry, I forgot about this one, I actually get the help text each time by running --help and copy pasting it into the file! The downside is that I have to remember to delete the globals...